activerecord/lib/active_record/core.rb

```diff
@@ -641,17 +641,22 @@ def strict_loading?
     #
     # ==== Parameters
     #
-    # * value - Boolean specifying whether to enable or disable strict loading.
-    # * mode - Symbol specifying strict loading mode. Defaults to :all. Using
-    #          :n_plus_one_only mode will only raise an error if an association
-    #          that will lead to an n plus one query is lazily loaded.
+    # * +value+ - Boolean specifying whether to enable or disable strict loading.
+    # * <tt>:mode</tt> - Symbol specifying strict loading mode. Defaults to :all. Using
+    #   :n_plus_one_only mode will only raise an error if an association that
+    #   will lead to an n plus one query is lazily loaded.
     #
-    # ==== Example
+    # ==== Examples
     #
     #   user = User.first
     #   user.strict_loading!(false) # => false
     #   user.comments
     #   => #<ActiveRecord::Associations::CollectionProxy>
+    #
+    #   user.strict_loading!(mode: :n_plus_one_only)
+    #   user.address.city # => "Tatooine"
+    #   user.comments
+    #   => ActiveRecord::StrictLoadingViolationError
```

Comment:
Should be `=> #<class>`? 🤔

Reply:
Not sure. The error class is raised, not returned.
MAybe we should do something like:
```suggestion
    #   raises ActiveRecord::StrictLoadingViolationError
```
We do that in other places as well:
https://github.com/rails/rails/blob/main/activerecord/lib/active_record/model_schema.rb#L352